### PR TITLE
Cumulus-496 Added integration test for discover-granules

### PR DESCRIPTION
**Pull request by kkelly51**
protocol: http

To test individually:
AWS_ACCOUNT_ID=# jasmine spec/discoverGranules/DiscoverGranulesSuccessSpec.js

### CODE REVIEW

File: spec/discoverGranules/DiscoverGranules.input.template.json [hunk 1]
```diff
@@ -0,0 +1,55 @@
+{
+  "cumulus_meta": {
+    "message_source": "sfn",
+    "state_machine": "",
+    "execution_name": ""
+  },
+  "meta": {
+    "workflow_name": "DiscoverGranules",
+    "stack": "",
+    "buckets": {
```
Comment (laurenfrederick) [inline spec/discoverGranules/DiscoverGranules.input.template.json:10]: Should this be ```"buckets": {
      "internal": "cumulus-test-sandbox-internal",
      "private": "cumulus-test-sandbox-private"
    },```?

File: spec/discoverGranules/DiscoverGranules.input.template.json [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+      "internal": "",
+      "private": ""
+    },
+    "provider": {
+      "id": "http_provider",
+      "host": "http://ec2-54-159-27-56.compute-1.amazonaws.com:3030",
```
Comment (laurenfrederick) [inline spec/discoverGranules/DiscoverGranules.input.template.json:16]: How did you set this up? I think we should document this. Also I am curious.

File: spec/discoverGranules/DiscoverGranules.input.template.json [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+      "protocol": "http"
+    },
+    "collection": {
+      "provider_path": "granules/fake_granules",
```
Comment (laurenfrederick) [inline spec/discoverGranules/DiscoverGranules.input.template.json:20]: Is this using the test-data package?

**Review by laurenfrederick — COMMENTED** [review]
Have some questions.

File: spec/discoverGranules/DiscoverGranules.input.template.json (hunk 1 above)
Comment (kkelly51) [reply spec/discoverGranules/DiscoverGranules.input.template.json:10]: I followed the trend of the other specs. They start blank in the ..input.template.json and then they get populated after running the test into the ...input.json. 
They get populated from the workflows.yml

File: spec/discoverGranules/DiscoverGranules.input.template.json (hunk 2 above)
Comment (kkelly51) [reply spec/discoverGranules/DiscoverGranules.input.template.json:16]: This is a temporary data endpoint set up (by @scisco), while we're waiting for https://bugs.earthdata.nasa.gov/browse/CUMULUS-465 to be unblocked.

File: spec/discoverGranules/DiscoverGranules.input.template.json (hunk 3 above)
Comment (kkelly51) [reply spec/discoverGranules/DiscoverGranules.input.template.json:20]: http://ec2-54-159-27-56.compute-1.amazonaws.com:3030/granules/fake_granules
This is the temporary endpoint but yes it set up similarly to the test-data package.

File: spec/discoverGranules/DiscoverGranules.input.template.json (hunk 1 above)
Comment (laurenfrederick) [reply spec/discoverGranules/DiscoverGranules.input.template.json:10]: Sorry! I had been looking at the generated file. My bad!

File: spec/discoverGranules/DiscoverGranules.input.template.json (hunk 2 above)
Comment (laurenfrederick) [reply spec/discoverGranules/DiscoverGranules.input.template.json:16]: Ok cool. Can you please update 465 to make sure that this endpoint gets updated when that ticket is completed?

**Review by laurenfrederick — APPROVED** [review]
Looks good! Just fix the build.

**Pull request merged**